test/msbuild-finder.js

```diff
@@ -186,10 +220,11 @@ describe('msbuild-finder', function () {
       var expectedResult = path.join(pathRoot, 'MSBuild', expectMSBuildVersion, 'Bin', 'MSBuild.exe');
 
       mock = this.sinon.mock(fs);
```

Comment:
You need to use these mocks in your other tests, so the tests will work as expected on other systems.